how-to/containerd-kata.md

```diff
@@ -185,14 +185,24 @@ The following configuration includes three runtime classes:
            SystemdCgroup = false
 ```
 
+From Containerd v1.2.4 and Kata v1.6.0, there is a new runtime option supported, by which you can specify a specific 
+kata configure file as below:
+
+```toml
+      [plugins.cri.containerd.runtimes.kata]
+         runtime_type = "io.containerd.kata.v2"
+	 [plugins.cri.containerd.runtimes.kata.options]
+	   ConfigPath = "/etc/kata-containers/config.toml"
+```
+
+This "ConfigPath" option is optional, if you doesn't specify it, shimv2 will first try to get the configure file from the environment
+variable "KATA_CONF_FILE", if it isn't set either, then shimv2 will search the configuration file in the default paths, 
+same as `kata-runtime` cli): `/etc/kata-containers/configuration.toml`, `/usr/share/defaults/kata-containers/configuration.toml`.
 Based on the discussion in [containerd/containerd#2916](https://github.com/containerd/containerd/pull/2916),
-the containerd configuration will have a field for runtime configuration files (likely to be`config_file`). However, 
-the field is not available yet, and the `shimv2` will search the configuration file in the default paths, same as 
-`kata-runtime` cli): `/etc/kata-containers/configuration.toml`, `/usr/share/defaults/kata-containers/configuration.toml`. 
-We will update this document once the `config_file` field is available.
 
-If you want to specify a configure file now, the following provides a workaround. The `shimv2` accepts an environment 
-variable, `KATA_CONF_FILE` for the config file path. Then, you can create a shell script with the following:
+If you use the Containerd and Kata's doesn't support this option and also want to specify a configure file, the following provides 
```

Comment:
/the following provides.../you can use the following workaround, since the `shimv2` accepts an environment variable, `KATA_CONF_FILE` for the config file path/